string subject
---
sem_server_ros/Triple[] result
